pkg/virt-handler/vm.go

```diff
@@ -98,6 +98,11 @@ type downwardMetricsManager interface {
 	StopServer(vmi *v1.VirtualMachineInstance)
 }
 
+type netSocketHandler interface {
```

Comment:
Following the split of the VM controller to three controllers, it makes sense to have a small interface per controller.